nes_bnf_komentarz.md

```diff
@@ -0,0 +1,26 @@
+<h2>
```

Comment:
Może następnym razem?
Przecież to działą. Nie szkoda jeszcze czasu/wysiłku na przeróbki?